rim/views.py

```diff
@@ -48,7 +84,6 @@ def get_queryset(self):
 
         return qset
```

Comment:
We should throw a `select_related()` on here. I think `latest_checkout__location` and `equipment_type` is all you need.